src/test/java/com/github/cameltooling/lsp/internal/hover/KameletHoverTest.java

```diff
@@ -45,10 +45,20 @@ void testProvideDocumentationOnHover() throws Exception {
 	void testProvideGenericDocumentationOnHoverOnUnknownKamelet() throws Exception {
 		CamelLanguageServer camelLanguageServer = initializeLanguageServer("<from uri=\"kamelet:unknown\" xmlns=\"http://camel.apache.org/schema/spring\"></from>\n");
 		
-		HoverParams hoverParams = new HoverParams(new TextDocumentIdentifier(DUMMY_URI+".xml"), new Position(0, 21));
+		HoverParams hoverParams = new HoverParams(new TextDocumentIdentifier(DUMMY_URI+".xml"), new Position(0, 19));
```

Comment:
why changing the value? Sounds like the hover should still work on position 19

Reply:
I simply put the start to after 'kamelet:'